src/shared/logger.ts

```diff
@@ -4,11 +4,11 @@ export class Logger {
   }
 
 
-  log(message, isError, container) {
+  log(message, isError, container?) {
```

Comment:
Another Typescript specific situation (this is why I prefer to use Typescript - if I knew how 😏 )